.github/workflows/update-screenshots.yml

```diff
@@ -49,17 +49,25 @@ jobs:
           DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -yq \
             libasound2-dev libudev-dev libxkbcommon-x11-0;
 
-      - name: install xvfb, llvmpipe and lavapipe
+      - name: Install xvfb, llvmpipe and lavapipe
         run: |
           sudo apt-get update -y -qq
           sudo add-apt-repository ppa:kisak/turtle -y
           sudo apt-get update
           sudo apt install -y xvfb libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
 
+      - name: Install oxipng
+        run: |
+          cargo install oxipng
```

Comment:
I'd use https://github.com/marketplace/actions/install-cargo-binstall and then run `cargo binstall` to not have to compile `oxipng` by hand

Reply:
not sure it's worth it, when the "taking the screenshots" part is already taking a few hours

Reply:
It comes at no cost to compression quality, so I don't see why not.

Reply:
It's additional complexity, maintenance cost and update management, for no benefit in compression quality

Reply:
Fair enough, I guess more people could chime in on the issue.

Reply:
after checking with other Bevy repos and conversation, we decided generally against using `cargo-binstall`. I think it should be kept the same and not changed just in a single place